src/institutions/Header.jsx

```diff
@@ -14,7 +14,9 @@ const InstitutionsHeader = ({ filingPeriod }) => {
       >
         <p>
           The beta {filingPeriod} HMDA Platform is available to upload, test,
-          and validate HMDA data.
+          and validate HMDA data. All test data uploaded during the beta
```

Comment:
`isBeta` is failing here so we're seeing the content from `beforeFilingPeriod`.

![beta](https://user-images.githubusercontent.com/2932572/48483015-415b6780-e7e0-11e8-8f2e-655ba258afe6.png)

I _think_ if you update [constants/dates.js](https://github.com/cfpb/hmda-platform-ui/blob/1ea1af930484e42bb229a39ac172e8f72689556d/src/constants/dates.js#L8) this will work.